### ISSUE TITLE
A11y_code.visualstudio.com(CELA-P1)_VS Code FAQ_Resize Text:At 125-175% zooming search section is not visible to the user

### ISSUE THREAD

**Issue by Amlan454** [microsoft/vscode-docs#2648]
Check out Keros!- Identify accessibility bugs before check-in and make bug fixing faster and easier.
**Environment Details:**
code.visualstudio.com/docs/supporting/faq
Browser: chrome , edge

**Note:
This issue is observed in below pages also:**
code.visualstudio.com/download
code.visualstudio.com/docs/supporting/faq
code.visualstudio.com/docs/setup/linux
code.visualstudio.com/docs/getstarted/settings
code.visualstudio.com/docs/python/python-tutorial
code.visualstudio.com/updates/v1_30
code.visualstudio.com/docs/editor/whyvscode
code.visualstudio.com/docs/getstarted/tips-and-tricks
code.visualstudio.com/updates/v1_29
code.visualstudio.com/docs/editor/integrated-terminal
code.visualstudio.com/docs/editor/debugging
code.visualstudio.com/docs/editor/versioncontrol

**Repro Steps:**
1)Hit the URL(code.visualstudio.com/docs/supporting/faq)and login by using appropriate credential
2)Zooming up to 175% and check whether the search section is visible or not

**Actual Result:**
At 125-175% zooming search section is not visible to the user

**Expected Result:**
Each every content should be visible to the user from 125-175% zooming

**User Impact:**
Low vision users will face difficulty to find out the control when they are navigating the page in magnifier mode

**Mas Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b69e5ccc-dde2-4406-b550-4deb38648f6d}

**Comment by Amlan454** [comment]
[Bug_2648_At 125 -175% zooming search section is not visible.pptx](https://github.com/microsoft/vscode-docs/files/3161886/Bug_2648_At.125.-175.zooming.search.section.is.not.visible.pptx)

**Comment by Amlan454** [comment]
![Bug_2648_1](https://user-images.githubusercontent.com/42638683/57602728-96a0e780-757d-11e9-9393-dddd0390b5d3.png)

**Comment by Amlan454** [comment]
![Bug_2648_1](https://user-images.githubusercontent.com/42638683/57692639-d8f12400-7664-11e9-8c4a-9cb4ffcfdb8f.png)

**Comment by Amlan454** [comment]
![Bug_2648_2](https://user-images.githubusercontent.com/42638683/57692712-1786de80-7665-11e9-841d-a57e57c3947c.png)

**Comment by auchenberg** [comment]
@misolori This is a design issue, as we don't scale the font-size in the navigation so on zoomed viewports, we are simply hiding the overflowing search box.

Any ideas?

**Comment by misolori** [comment]
@auchenberg we could try to make the search field responsive and always show it (on smaller screens we could add an expand on click):

**150% Zoom**
![image](https://user-images.githubusercontent.com/35271042/59299128-cc9abf80-8c40-11e9-9364-9bb67cdb44bf.png)

**175% Zoom**
![image](https://user-images.githubusercontent.com/35271042/59299275-1683a580-8c41-11e9-9f8c-77f401819034.png)

**175% Zoom + Expanded**
![image](https://user-images.githubusercontent.com/35271042/59299821-79c20780-8c42-11e9-9d53-ca144ebd149f.png)

**Comment by auchenberg** [comment]
@misolori How about this. When zoomed the input field from search disappears, but the search icon and download button says as links. Clicking the search icon takes you to https://code.visualstudio.com/search

**100%:**
<img width="1772" alt="Screen Shot 2019-06-23 at 12 45 28 PM" src="https://user-images.githubusercontent.com/173559/59981197-0363c480-95b5-11e9-9e53-bbd45701c620.png">

**200%**
<img width="1772" alt="Screen Shot 2019-06-23 at 12 45 35 PM" src="https://user-images.githubusercontent.com/173559/59981204-09f23c00-95b5-11e9-8b82-73ae5b2609ef.png">

**Comment by fsteffi** [comment]
GitHubTags:#CAITestPass_June’19;
